src/updateOutdatedPullsTemplates.ts

```diff
@@ -19,6 +19,8 @@ if (import.meta.main) {
 export async function updateOutdatedPullsTemplates() {
   const pyproject = await readTemplate("add-toml.md");
   const publishcr = await readTemplate("add-action.md");
+  const toml = await readTemplate("outdated_toml")
```

Comment:
const toml = await readTemplate("outdated_toml")

I guess filename should be  "templates/update-toml-license.md" ?